csharp-intermediate-topics/ConstVsReadonlyInCSharp/ConstVsReadonlyInCSharp/TaxCalculator.cs

```diff
@@ -0,0 +1,25 @@
+﻿namespace ConstVsReadonlyInCSharp
+{
+    public class TaxCalculator
```

Comment:
As a general rule in C#, whenever we are dealing with currency, we should use the decimal type rather than the double type due to floating point errors. I know this is a simple example, but whenever possible we should reflect industry standards and practices